closure/goog/math/integer.js

```diff
@@ -566,6 +566,60 @@ goog.math.Integer.carry16_ = function(bits, index) {
   }
 };
 
+/**
+ * Returns this Integer divided by the given one.
+ * NOTE: Both this and the given Integer MUST be positive.
+ * @param {goog.math.Integer} other The Integer to divide this by.
```

Comment:
> is nullable by default

does this mean all the code in the codebase is supposed to allow null arguments, or that I've accidentally indicated `other` can be null, or something else?